flatcontent/tests/test_flatcontent.py

```diff
@@ -55,6 +55,19 @@ def test_get_not_found(self):
         # Test the flatcontent doesn't exist
         eq_('', FlatContent.get('not-found'))
 
+    def test_get_with_context(self):
+        flatcontent = FlatContentFactory.create(
+            slug='test-tpl', site=None, content='Hi {{ name }}')
+
+        assert FlatContent.get('test-tpl') == 'Hi '
+
+        # Clear the cache
+        flatcontent.save()
```

Comment:
@brad Maybe we should add the context to the cache key? It seems contrary to expectation that the caching would cause the template to render incorrectly in some cases. I think we have flatcontent that uses usernames in the context, so I can image a case where users would get content with the wrong username in it.